pkg/kubelet/prober/common_test.go

```diff
@@ -108,6 +108,7 @@ func newTestManager() *manager {
 		nil, // runner
 		refManager,
 		&record.FakeRecorder{},
+		"",
```

Comment:
add some comment about the meaning of this empty string?

Reply:
Added comment. Thanks.